lib/questions.rb

```diff
@@ -155,30 +185,38 @@ def get_domain_name_from_email_address(email)
 # 'the lion the witch and the wardrobe' becomes
 # 'The Lion the Witch and the Wardrobe'
 def titleize_a_string(string)
+  string.split(" ").map {|word| word.capitalize unless $nocaps.include?(word)}.join(" ")
```

Comment:
Line is too long. [88/80]<br>Space between { and | missing.<br>Space missing inside }.